x-pack/plugins/observability/public/components/shared/exploratory_view/series_viewer/columns/filter_expanded.test.tsx

```diff
@@ -96,13 +104,15 @@ describe('FilterExpanded', function () {
       { initSeries }
     );
 
-    fireEvent.click(screen.getByText('Browser Family'));
+    await waitFor(() => {
```

Comment:
Thanks for fixing these tests. Can you let me know the error you were running into, for my edification?

Reply:
Many of these components do data fetching when mounted, so they get the [`act` error described in this blog post](https://davidwcai.medium.com/react-testing-library-and-the-not-wrapped-in-act-errors-491a5629193b) (see Case 1 there.)

I didn't have any concern about these errors specifically, just want to be running Jest without seeing a bunch of warnings.